app/assets/javascripts/project.js

```diff
@@ -55,16 +55,6 @@ document.addEventListener("turbolinks:load", function() {
   })
 })
 
-window.addEventListener("load", () => {
-  const links = document.querySelectorAll(".project-table__row--story a.delete");
```

Comment:
@arielj Any idea why the original method did a `querySelectorAll`?

Reply:
This was attaching a callback to all the `Delete` buttons for each individual story, so it had to loop through all the elements